hack/run-obs-e2e-test.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
this file is unnecessary, please remove it

Reply:
this file will be deleted after e2e-obs shell checked.